tests/framework/utils.py

```diff
@@ -515,3 +516,26 @@ def compare_versions(first, second):
         return first[1] - second[1]
 
     return first[0] - second[0]
+
+
+class ResultsFileDumper:
+    """Class responsible with outputting test results to files."""
+
+    def __init__(self, test_name: str):
+        """Initialize the instance."""
+        self._root_path = defs.TEST_RESULTS_DIR
+        self._path = self._root_path / test_name
+
+        # Create the root directory, if it doesn't exist.
+        os.makedirs(self._root_path, exist_ok=True)
```

Comment:
nit: since `_root_path` is a Pathlib.Path instance you can run the built-in mkdir function https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir

Reply:
done